### ISSUE TITLE
Buttons are hidden by default on small screens

### ISSUE THREAD

**Issue by ddikman** [alexwforsythe/code-blocks#60]
Not sure if I didn't get how to use it or if it didn't work. The block where I expect I should write code simply was grey.

![image](https://user-images.githubusercontent.com/8461739/36060856-f407985c-0e94-11e8-8842-ce16a47e7fcd.png)

Best regards,
David

**Comment by alexwforsythe** [comment]
Hi David. The grey box is just a preview area. To use the add-on:
1. Select some text in your document
2. Click **Preview** to preview the selected theme (optional)
3. Click **Format** to replace the selected text in your document with the formatted code block

**Comment by ddikman** [comment]
I see, thank you for the response! Maybe I'm being obliviously stupid here but, where does the preview button show up?

**Comment by alexwforsythe** [comment]
Oh, I didn't even notice that the buttons are missing in your screenshot! That's probably a bug. Could you try reinstalling the add-on? If they still don't show up, can you tell me what browser you're using?

**Comment by ddikman** [comment]
Hi mate,
Thanks for getting back to me.

Tried reinstalling but to no avail. However :) when playing around with the developer tools having it up on a smaller screen and by accident scrolling I found it :)

![image](https://user-images.githubusercontent.com/8461739/36148739-6946a992-1100-11e8-845c-a9bb9deeac38.png)

Chrome sneakily hides away that scrollbar hiding the button in low resolution screens (I'm on a 11" macbook air).

Could fix the button at the bottom of the panel or so but no biggie :) if I get to annoyed with it I can try and fork a fix. 

Once working it looks real nice though :)

Thanks again.

/ David
